x/docs/md/cli/linux.md

```diff
@@ -1,7 +1,7 @@
 
 ## Linux
 
-View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases/tag/v2.3.0).
+View the CLI releases to [choose and download the appropriate package for your operating system](https://github.com/exercism/cli/releases).
```

Comment:
Yeah, I think changing the text is the right call. If we can get them as close to where they need to be as possible, that seems like a good thing.